en/docs/learn/examples/jms-examples/consume-produce-jms.md

```diff
@@ -4,20 +4,18 @@ This section describes how to configure WSO2 Micro Integrator to work as a JMS-t
 
 ## Synapse configuration
 
-Given below is the synapse configuration of the proxy service that mediates the above use case. Note that you need to update the JMS connection URL according to your broker as explained below. See the instructions on how to [build and run](#build-and-run) this example.
+Given below is the synapse configuration of the proxy service that mediates the above use case. See the instructions on how to [build and run](#build-and-run) this example.
 
 ```xml
-<proxy name="StockQuoteProxy" transports="jms">
+<proxy name="StockQuoteProxy" startOnLoad="true" transports="jms" xmlns="http://ws.apache.org/ns/synapse">
     <target>
-        <inSequence>
-            <property action="set" name="OUT_ONLY" value="true"/>
-            <send>
-                <endpoint>
```

Comment:
Why the endpoint call is removed?

Reply:
The scenario of this page is to listen to a JMS queue, consume its messages, and then publish those messages to another JMS queue. 
The endpoint call will only publish the messages the proxy service receives when called to the provided queue(not configured to listen to JMS). To adhere to the actual use case, I changed the proxy service to listen to JMS and then publish the messages it receives to another queue

Reply:
This is fixed now